docs/guides/web-servers/caddy/how-to-install-and-configure-caddy-on-centos-8/index.md

```diff
@@ -36,158 +36,105 @@ aliases: ['/web-servers/caddy/how-to-install-and-configure-caddy-on-centos-8/']
-    {{</ note >}}
+        v2.4.3 h1:Y1FaV2N4WO3rBqxSYA8UZsZTQdN+PwcoOcAiZTM8C0I=
 
-1. Add Caddy to your system's `$PATH`.
+{{< caution >}}
+Caddy has recently changed their [license](https://caddyserver.com/products/licenses). Please read over the license agreement to ensure that you are not violating the license with your project. To use Caddy without a commercial license, you may need to [compile from source](/docs/web-servers/caddy/compile-caddy-from-source).
+{{</ caution >}}
 
-        sudo echo 'export PATH=/usr/local/bin/caddy:$PATH' | sudo tee /etc/profile.d/caddy.sh
+## Allow HTTP and HTTPS Connections
 
-1. Reload your system's profile or log out and SSH back into your Linode.
+Caddy serves websites using HTTP and HTTPS protocols, so you need to allow access to the ports 80, and 443.
 
-        . /etc/profile
+        sudo firewall-cmd --permanent --zone=public --add-service=http
+        sudo firewall-cmd --permanent --zone=public --add-service=https
+        sudo firewall-cmd --reload
+
+## Add Web Content
+
+1.  Set up a home directory, **web root**, for your website:
+
+        sudo mkdir -p /var/www/html/example.com
+
+1. Use SELinux’s `chcon` command to change the file security context for web content:
 
-    {{< note >}}
-You can verify that the Caddy executable is in your system's `$PATH` with the following command:
+        sudo chcon -t httpd_sys_content_t /var/www/example.com -R
+        sudo chcon -t httpd_sys_rw_content_t /var/www/example.com -R
 
-    echo $PATH
+1.  Create a test page:
 
-The output should include the location of your Caddy executable:
+        echo '<!doctype html><head><title>Caddy Test Page</title></head><body><h1>Hello, World!</h1></body></html>' > /var/www/html/example.com/index.html
 
-    /usr/local/bin/caddy:/home/example_user/.local/bin:/home/example_user/bin:/usr/local/bin/caddy:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin
+## Configure the Caddyfile
 
-    {{</ note >}}
+Add your hostname and web root to the Caddy configuration. Use an editor of your choice and replace `:80` with your domain name. Set the root directory of the site to `/var/www/html/example.com` Replace `example.com` with your site's domain name:
 
-1. Install Caddy as a systemd service.
+{{< file "/etc/caddy/Caddyfile" caddy >}}
+example.com {
+root /var/www/html/example.com
+}
+{{< /file >}}
 
-        sudo env "PATH=$PATH" caddy -service install
+## Start and Enable the Caddy Service
 
 1. Temporarily set SELinux to permissive mode in order to start the Caddy service.
 
         sudo setenforce 0
 
-1. Start the Caddy service:
+1.  Enable the Caddy service:
 
         sudo systemctl start caddy
 
 1. Verify that the service is active:
 
         sudo systemctl status caddy
 
-    You should see a similar output:
+    An output similar to the following appears:
```

Comment:
It should be noted that `sudo systemctl status caddy` _truncates log lines_, so `journalctl -u caddy --no-pager | less +G` is what we recommend to actually see the latest logs without truncation.

Reply:
I don't see any change to solve this one?

Reply:
This was updated as a note.